src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers.Adapters/XmlExporter.cs

```diff
@@ -0,0 +1,198 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using Java.Interop.Tools.JavaCallableWrappers.CallableWrapperMembers;
+using Java.Interop.Tools.JavaCallableWrappers.Extensions;
+
+namespace Java.Interop.Tools.JavaCallableWrappers.Adapters;
+
+public static class XmlExporter
+{
+	static XmlWriterSettings settings = new XmlWriterSettings {
+		Indent = true,
+		NewLineOnAttributes = false,
+		OmitXmlDeclaration = true,
+	};
+
+	public static void Export (string filename, IEnumerable<CallableWrapperType> types, bool wasScanned)
+	{
+		using (var sw = new StreamWriter (filename, false, Encoding.UTF8))
+			Export (sw, types, wasScanned);
+	}
+
+	public static void Export (TextWriter sw, IEnumerable<CallableWrapperType> types, bool wasScanned)
+	{
+		using (var xml = XmlWriter.Create (sw, settings))
+			Export (xml, types, wasScanned);
+	}
+
+	public static void Export (XmlWriter xml, IEnumerable<CallableWrapperType> types, bool wasScanned)
+	{
+		ExportTypes (xml, types, wasScanned);
+	}
+
+	static void ExportTypes (XmlWriter xml, IEnumerable<CallableWrapperType> types, bool wasScanned)
+	{
+		xml.WriteStartElement ("types");
+		xml.WriteAttributeString ("was_scanned", wasScanned.ToString ());
+
+		foreach (var type in types)
+			ExportType (xml, type);
+
+		xml.WriteEndElement ();
+	}
+
+	public static void ExportType (XmlWriter xml, CallableWrapperType type)
+	{
+		xml.WriteStartElement ("type");
```

Comment:
…and now it's bike shedding time (?).

Should this "rhyme with" Our Favorite XML Format™, `class-parse` output?  That's what `tools/java-source-utils` does when extracting Javadoc documentation, e.g. https://github.com/dotnet/java-interop/blob/03cb8e258735a5e0b8a5c703832639fb6b65f3c7/tools/java-source-utils/src/test/resources/com/microsoft/android/DemoInfo.xml#L1-L27

Perhaps something like:

```xml
<api api-source="jcw">
  <package name="crc64…">
    <class name="ExampleType" abstract="true" extends="java.lang.Object"
        jcw.HasExport="false" jcw.IsApplication="false" …>
```